registration-processor/core-processor/registration-processor-abis-middleware-stage/src/test/java/io/mosip/registartion/processor/abis/middleware/stage/AbisMiddleWareStageTest.java

```diff
@@ -337,21 +343,24 @@ public void testVariousScenarious() throws RegistrationProcessorCheckedException
 		Mockito.when(packetInfoManager.getReferenceIdByRid(Mockito.anyString())).thenReturn(null);
 		stage.deployVerticle();
 		stage.process(dto);
-		assertFalse(dto.getIsValid());
+		assertTrue(dto.getIsValid());
+		assertTrue(dto.getInternalError());

```

Comment:
Why all below 4 changes are retriable error cases ?

Reply:
whenever we catch any exception in middleware stage, it is setting latest Transaction code to REPROCESS, that's why it is retriable...